File: WSPTools/App/WSPBuilderTemplates/VBTemplates/Item/VB_SequentialWorkFlowFeature/NewWorkflow.vb
Imports System
Imports System.ComponentModel
Imports System.ComponentModel.Design
Imports System.Collections
Imports System.Drawing
Imports System.Workflow.ComponentModel.Compiler
Imports System.Workflow.ComponentModel.Serialization
Imports System.Workflow.ComponentModel
Imports System.Workflow.ComponentModel.Design
Imports System.Workflow.Runtime
Imports System.Workflow.Activities
Imports System.Workflow.Activities.Rules
Imports Microsoft.SharePoint
Imports Microsoft.SharePoint.Workflow
Imports Microsoft.SharePoint.WorkflowActions
Imports Microsoft.Office.Workflow.Utility

Namespace $DefaultNamespace$
    Partial Public NotInheritable Class $rootname$
        Inherits StateMachineWorkflowActivity
        Public Sub New()
            InitializeComponent()
        End Sub
        Public workflowId As Guid = Nothing
        Public workflowProperties As SPWorkflowActivationProperties = New Microsoft.SharePoint.Workflow.SPWorkflowActivationProperties()
    End Class

End Namespace

File: WSPTools/App/WSPBuilderTemplates/Templates/Item/StateWorkFlowFeature/NewWorkflow.cs
using System;
using System.ComponentModel;
using System.ComponentModel.Design;
using System.Collections;
using System.Drawing;
using System.Workflow.ComponentModel.Compiler;
using System.Workflow.ComponentModel.Serialization;
using System.Workflow.ComponentModel;
using System.Workflow.ComponentModel.Design;
using System.Workflow.Runtime;
using System.Workflow.Activities;
using System.Workflow.Activities.Rules;
using Microsoft.SharePoint;
using Microsoft.SharePoint.Workflow;
using Microsoft.SharePoint.WorkflowActions;
using Microsoft.Office.Workflow.Utility;

namespace $DefaultNamespace$.Workflow
{
	public sealed partial class $rootname$ : StateMachineWorkflowActivity
	{
		public $rootname$()
		{
			InitializeComponent();
		}

        public SPWorkflowActivationProperties workflowProperties = new Microsoft.SharePoint.Workflow.SPWorkflowActivationProperties();
	}

}
